bin/http-server

```diff
@@ -255,24 +255,23 @@ function listen(port) {
 
     // Spacing before logs
     if (!argv.s) logger.info();
-  });
-}
 
-if (process.platform === 'win32') {
-  require('readline').createInterface({
-    input: process.stdin,
-    output: process.stdout
-  }).on('SIGINT', function () {
-    process.emit('SIGINT');
-  });
-}
```

Comment:
Moved the block for Windows to the bottom, so that the reading order becomes natural.